CONTRIBUTING.MD

```diff
@@ -34,6 +34,29 @@ Example:
 }
 ```
 
+#### Linking local changes from `@faststore/api`
+
+`graphql-js` package is cumbersome when using `yarn link` because it requires only one instance of the package and there are two.
```

Comment:
I think a more generic sentence fits better since `graphql-js` is not the only problem.

```suggestion
Sometimes symbolic links just aren't enough (e.g. dealing with local `graphql-js` package is cumbersome when using `yarn link` because it requires only one instance of the package and there are two).
```